### PR TITLE
Add New Repo instructions

### PR DESCRIPTION
**Pull request by ghost**
# Description

Add first instructions how to create new repos.

# How Has This Been Tested?

- [x] [Bootstrap](https://github.com/qbeyond/terraform-google-landing-zone-bootstrap) @QBY-ThimoLimpert  
- [x] [Service Account Module](https://github.com/qbeyond/terraform-google-service-account) @PLN-Qbeyond 

# Checklist:

- [x] I have run tests and documented them above
- [x] I have performed a self-review of my own code
- [x] I have updated the documentation

### CODE REVIEW

File: NOTICE.template [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Cloud-Foundation-Folder-Module
+
+Copyright 2022 q.beyond AG
```
Comment (QBYMKPI) [inline NOTICE.template]: Doesn't this have to be Copyright 2023 q.beyond AG?

**Review by QBYMKPI — CHANGES_REQUESTED** [review]

**Review by QBYMKPI — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,122 @@
...
+
+**You must add a prominent notice to every file you change!** Any file should already contain an Boilerplate header. If not you can copy it from `LICENSE`. Add the appropriate Notice above the existing information `Copyright 2023 q.beyond AG`. When changing other files without a Copyright notice, please add it and add the q.beyond claim accorsingly.
+
+When you finished fixing all references to other directoy, test the module thourougly. 
+
+When everything is working as expected copy the files `.terraform-docs.yml` and `.github/workflows/docs.yaml` from the [terraform-module-template](https://github.com/qbeyond/terraform-module-template). Run the auto docu once to check the repository. Afterwards read to the `README.md` and change where needed. At least add a reference `This stage is part of the [google landing zone modules](https://github.com/qbeyond/terraform-google-landing-zone).`
```
Comment (schnaddy) [inline README.md]:
```suggestion
When everything is working as expected copy the files `.terraform-docs.yml` and `.github/workflows/docs.yaml` from the [terraform-module-template](https://github.com/qbeyond/terraform-module-template). Run the auto docu once to check the repository. Afterwards read to the `README.md` and change where needed. At least add a reference `This stage is part of the [google landing zone modules](https://github.com/qbeyond/terraform-google-landing-zone).`
```
```suggestion
When everything is working as expected copy the files `.terraform-docs.yml` and `.github/workflows/docs.yaml` from the [terraform-module-template](https://github.com/qbeyond/terraform-module-template). Run the auto docu once to check the repository. Afterwards head to the `README.md` and change where needed. Lastly add a reference `This stage is part of the [google landing zone modules](https://github.com/qbeyond/terraform-google-landing-zone).`
```

**Review by schnaddy — CHANGES_REQUESTED** [review]
small corrections

**Review by schnaddy — APPROVED** [review]
All looks good can't find any Typos or other errors. Nice Job!

**Pull request merged**